README.md

```diff
@@ -224,14 +226,21 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `AUTH`   | used to send authentication challenges                  | [42](42.md) |
 | `COUNT`  | used to send requested event counts to clients          | [45](45.md) |
 
-## Standardized Tags
+## Tags
+
+### Standardized Tags
```

Comment:
```suggestion
### Common Tags

These tags are widely used across many different event kinds with the exact same structure.
```

Reply:
I think the meaning of these tags is explained in NIP-01.

https://github.com/nostr-protocol/nips/blob/d251ca0da5372ba0ae0df76754d96068266a6778/01.md?plain=1#L76

Reply:
Oh! Good point. 😃

Reply:
Then I would call them `Standard` rather than `Standardized`

Reply:
Agreed.